src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -67,11 +66,11 @@ public BattleshipModel() {
         playerCivilianShips = new CivilianShip[2];
 
 
-        computer_aircraftCarrier = (MilitaryShip) placeEnemyShip("Computer_AircraftCarrier", 5);
-        computer_battleship = (MilitaryShip) placeEnemyShip("Computer_Battleship",4);
-        computer_submarine = (MilitaryShip) placeEnemyShip("Computer_Submarine",2);
-        computer_clipper = (CivilianShip) placeEnemyShip("Computer_Clipper",3);
-        computer_dinghy = (CivilianShip) placeEnemyShip("Computer_Dinghy",1);
+        computer_aircraftCarrier = (MilitaryShip) placeEnemyShip("Computer_AircraftCarrier", 5,true);
+        computer_battleship = (MilitaryShip) placeEnemyShip("Computer_Battleship",4,true);
+        computer_submarine = (MilitaryShip) placeEnemyShip("Computer_Submarine",2,true);
+        computer_clipper = (CivilianShip) placeEnemyShip("Computer_Clipper",3,true);
+        computer_dinghy = (CivilianShip) placeEnemyShip("Computer_Dinghy",1,true);
```

Comment:
Will this true value be a difficulty boolean in the future? As it is the difficulty will always be set to easy.